blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -36,15 +44,20 @@
 
     protected final User user;
 
+    private final BlueOrganization organization;
+    private final AccessControlled organizationBase;
+
     private final Reachable parent;
-    public UserImpl(User user, Reachable parent) {
+
+    public UserImpl(@Nonnull BlueOrganization organization, @Nonnull User user, Reachable parent) {
```

Comment:
... but I understand the complication with the nature of how this API returns JSON from the domain objects.